src/capture-eye.ts

```diff
@@ -91,18 +91,17 @@ export class CaptureEye extends LitElement {
   }
 
   private buttonTemplate() {
-    const isMobile = hasTouchScreen();
     return this.nid
       ? html`
           <div
             class="capture-eye-button ${this.visibility ===
-              Constant.visibility.always || isMobile
+              Constant.visibility.always || isMobile()
               ? 'full-visibility'
               : ''}"
             @click=${this.openEye}
           >
             <img
-              src=${isMobile
+              src=${isMobile()
```

Comment:
how about using a constant like before so we don't have to call the function twice?